lang/python/2to3/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=2to3
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+
```

Comment:
If found nowhere the license details, in fact the main repository show a no license; but all Debian softwares are GPL...
I will check on Debian main site.

I propose to add the GIT-LINK (ref) : https://salsa.debian.org/cpython-team/python3-defaults

Reply:
I don’t have a strong opinion on whether to include the script locally in the package or to download from upstream, since the script is so small and very unlikely to change.

If you want to download from upstream, you can download a [tagged release](https://salsa.debian.org/cpython-team/python3-defaults/-/tags) using the archive links.

Reply:
I think it is out of purpose to get a so huge code for a so small script.
Which, as you said, will be very unlikely to change !

Do we need to keep the license (COPYRIGHT) file ?

Reply:
COPYRIGHT file removed.
script included.